plugins-base/XSFeatureStatus.py

```diff
@@ -18,6 +18,20 @@
     
 from XSConsoleStandard import *
 
+def is_master():
+    master = False
+    try:
```

Comment:
Python is not my main language but is this not a textbook case for `with open() as file`?

Reply:
Yes, better code indeed.
I'm not sure about using a `try` around the `open` as well. if `pool.conf` can't be read, the exception should be raised no?